backend/src/main/resources/application-aws.yml

```
@@ -47,4 +47,9 @@ jwt:
     secretKey: ${JWT_SECRET}
 
 weather:
-  encoding-key: ${WEATHER_KEY}
\ No newline at end of file
+  encoding-key: ${WEATHER_KEY}
+
+ffmpeg:
+  path: /usr/bin/ffmpeg
+ffprobe:
+  path: /usr/bin/ffprobe
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)